goship.go

```
@@ -662,7 +675,7 @@ func PostPivotalComment(id string, m string, piv *goship.PivotalConfiguration) (
 func DeployPage(w http.ResponseWriter, r *http.Request) {
 	p := r.FormValue("project")
 	env := r.FormValue("environment")
-	user := r.FormValue("user")
+	user, _ := getUser(r)
```

Swallowing an error here :walking: